### ISSUE TITLE
[FE] - Auth Profile - Address Change Messaging - Make address update modal content swappable

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#37842]
## Background

There is an edge case where a Veteran does not have a mailing address stored in their VA.gov Profile. If they update their home address, we would like the modal to be smart enough to display a slightly different message from what is shown below when the mailing address is null/empty.  Currently, the code is not written to allow that text to be conditional. 

The following Sketch mockups shows how the message should appear and what it should say: Desktop: https://www.sketch.com/s/59857eb5-d9f9-4145-99d3-d9a1de2d0655/a/25K3vb5

Mobile: https://www.sketch.com/s/59857eb5-d9f9-4145-99d3-d9a1de2d0655/a/09Vb1eV

## Tasks
- [x] Create logic that allows the text within the home address update confirmation to be conditional
- [x] Conduct all required testing
- [x] Update PM when the change is on Staging for validation

## Acceptance Criteria
- [x] Create logic that allows the text within the home address update confirmation to be conditional
- [x] Conduct all required testing
- [x] Update PM when the change is on Staging for validation

## How to configure this issue
- Include practice area in title -- e.g., [Design], [FE], [BE]
- Include project/initiative name - e.g., Auth Profile: Address Change Messaging
- Add label for practice area (frontend, backend, design)
- Add label for project if applicable
- Assign to a Profile team member
- Associate with an Epic if applicable
- If creating for yourself: estimate work if enough info exists to do so

**Comment by MarciMcGuireGCIO** [comment]
@adamwhitlock1 can you please take a look at this issue and estimate when you have a few min? Thank you!

**Comment by MarciMcGuireGCIO** [comment]
@adamwhitlock1 - I added a link to Christina's mockup in the description at the top.

**Comment by adamwhitlock1** [comment]
@SKasimow and @Samara-Strauss I have completed this and have it up on staging for validation. Do we have any test users that don't have a mailing address at this point? I tried a few but they all seemed to have at least a mailing address or else they weren't LOA3 so didnt show the section at all.

This is what the modal looks like when no address is present, but I'm unsure if there would be an easy way to test this fully within staging.

![Screen Shot 2022-04-08 at 10 31 09 AM](https://user-images.githubusercontent.com/8332986/162484301-039a13bc-2ee6-4a06-ba7c-4b58580a38de.png)

Also in a related matter, I did open a ticket to have the margin issues addressed within modal (the excess margin on the right side) and the ticket for that is https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/693
